PopPUNK/__main__.py

```diff
@@ -61,46 +26,18 @@ def get_options():
     import argparse
 
     parser = argparse.ArgumentParser(description='PopPUNK (POPulation Partitioning Using Nucleotide Kmers)',
-                                     prog='PopPUNK')
+                                     prog='poppunk')
 
     modeGroup = parser.add_argument_group('Mode of operation')
     mode = modeGroup.add_mutually_exclusive_group(required=True)
```

Comment:
I am sad to lose `--easy-run` - do you think there's a role for a script that runs a similar pipeline for bioinformatician convenience?

Reply:
We can certainly add a script to pipeline those commands, but I think it made the control flow through main unnecessarily painful to follow. Perhaps now it could even be done with snakemake